.github/workflows/android.yml

```diff
@@ -23,8 +23,8 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-  build-llm-demo:
-    name: build-llm-demo
+  build-aar:
```

Comment:
It's better to refactor `build-arr` and `upload-release-aar` to a separate dispatchable workflow that:

1. android.yml can call
2. and we can manually dispatch it to build/upload Android release artifacts

However, that could come in a later PR if you prefer